src/core/tsdf_submap_collection.cpp

```diff
@@ -147,14 +202,14 @@ void TsdfSubmapCollection::getSubMapPoses(
   CHECK_NOTNULL(submap_poses_ptr);
   // Extracting transforms
   submap_poses_ptr->clear();
-  submap_poses_ptr->reserve(tsdf_sub_maps_.size());
-  for (TsdfSubmap::ConstPtr tsdf_sub_map : tsdf_sub_maps_) {
-    submap_poses_ptr->push_back(tsdf_sub_map->getPose());
+  submap_poses_ptr->reserve(id_to_submap_.size());
+  for (const auto& id_submap_pair : id_to_submap_) {
+    submap_poses_ptr->push_back((id_submap_pair.second)->getPose());
   }
 }
 
-bool TsdfSubmapCollection::getAssociatedTsdfSubMapID(
-    const SubmapID submap_id, SubmapID *submap_id_ptr) const {
+/*bool TsdfSubmapCollection::getAssociatedTsdfSubMapID(
```

Comment:
If commented out, delete?

Reply:
Yeah this is waiting to be moved to manifold mapping which requires some additional functionality that doesn't make sense here. It's hideous, but I'm just leaving it commented out here so I don't forget.